utHelper.py

```diff
@@ -0,0 +1,372 @@
+
+        Returns:
+            str: The session output up to and including the found message.
+        """
+        self.log.debug("waitForSessionMessage([{}])".format(message).format(dut))
+
+        activeDevice = self.devices.getDevice(device)
+        session = activeDevice.getConsoleSession()
+
+        try:
+            # Read from the session until the message is found
+            string = session.read_until(message)
+
+            # Check if the message was actually found
+            findStringLocation = string.find(message)
+            if findStringLocation == -1:
+                self.log.error("Could not find string: {}, raise an exception".format(message))
+                raise Exception("Raise an exception")  # Raise a generic exception
+                return False  # This line is unreachable due to the exception
+
+        except Exception as e:
+            self.log.error(e)
+            raise Exception('waitForSessionMessage session message - {} failed'.format(message))
+
+        return string
+
+    def writeMessageToDeviceSession(self, message, device="dut"):
+        """
+        Writes a message to the console session of the specified device.
+
+        Args:
+            message (str): The message to write.
+            device (str, optional): The device to write the message to (default: "dut").
+        """
+        self.log.step("testControl.writeMessageToSession({})".format(message.strip()))
+
+        activeDevice = self.devices.getDevice(device)
+        session = activeDevice.getConsoleSession()
+        session.write(message)
+
+    ## Device file operations
+
+    def createDirectoryOnDevice(self, dirPath, device="dut"):
+        """
+        Creates a directory on the connected target device.
+
+        Args:
+            dirPath (str): The path of the directory to be created.
+            device (str, optional): The device on which to create the directory (default: "dut").
+        """
+        activeDevice = self.devices.getDevice(device)
+        session = activeDevice.getConsoleSession()
+        self.log.info("dirPath:[%s]", dirPath)
+        session.write("mkdir " + dirPath)  # Send the 'mkdir' command to create the directory
+        session.write("\n")  # Send a newline to execute the command
+
+    def mountFolderOnDevice(self, devPath, mntPath, device="dut"):
+        """
+        Mounts a device path (e.g., a USB drive) to a mount point on the target device.
+
+        Args:
+            devPath (str): The device path to mount (e.g., /dev/sda1).
+            mntPath (str): The mount point on the target device.
+            device (str, optional): The device on which to mount the folder (default: "dut").
+        """
+        self.log.stepMessage("Mount USB " + devPath + " to " + mntPath + " path")
+        self.writeMessageToDeviceSession("mount " + devPath + " " + mntPath, device=device)  # Send the 'mount' command
+        self.writeMessageToDeviceSession("\n", device=device)  # Send a newline
```

Comment:
Do we want this? We're not going to have USB mounts on the racks (though we might need them for bootloader), so we'd want this to be just be a copy to device method that works out how to do based on a config option?